tools/codegen/model.py

```diff
@@ -125,6 +125,29 @@ class NativeFunction:
     # in terms of the out kernel referenced by the string here.
     structured_delegate: Optional['OperatorName']
 
+    # Note [Abstract ATen methods]
+    # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    # An abstract ATen method is one whose dispatch differs between
+    # types.  These are implemented in derived types (with a
+    # standard (throwing) definition in Type).  A concrete ATen
+    # method is one which has the same dispatch for all types;
+    # we just implement it in the base Type.  This is exposed
+    # in Declarations.yaml via a field named 'abstract'.
+    #
+    # Although this is what we have historically exposed, it is
+    # actually not all that useful for end users, who are also interested
+    # whether or not there is an explicit entry in derivatives.yaml
+    # for the entry or not (as this affects whether or not the operation is
+    # overrideable or not.)  Once this all gets cleaned up, this
+    # property will be obsolete.
```

Comment:
So.... is it or is it not useful for end users? :)

Reply:
I wasn't entirely sure whether the original comment was talking about external users of Declarations.yaml (it's kinda useful for autograd codegen) - I'll remove this paragraph.